ComSemApp/student/views.py

```diff
@@ -673,3 +679,44 @@ def form_invalid(self, form):
     def form_valid(self, form):
         form.save()
         return JsonResponse({}, status=200)
+
+#transcribes the audio received from ajax call in ComSemRecording-opus.js
+#written by Nate Kirsch and Jalen Tacsiat
+def transcribe(request):
```

Comment:
it looks like this view and the one in teachers/views.py are identical. Could we move the logic that's common to the two into a helper function? Maybe in ComSemApp.utils.py?